docs/getting-started.md

```diff
@@ -74,7 +74,75 @@ To create a new headless WordPress app using SnapWP, follow these steps:
 
 ### Manual Installation
 
-@todo
+1. Install npm packages in your Next App
+
+```bash
+npm i @snapwp/blocks @snapwp/core @snapwp/next @snapwp/query
+```
+
+2. Copy the .env contents from `Dashboard > WPGraphQL > Settings > SnapWP Helper` from the WP server and create an `.env` file in the project root.
+
+3. Create `snapwp.config.ts` in the project root with the following content.
+
+```typescript
+import type { SnapWPConfig } from '@snapwp/core/config';
+
+const config: SnapWPConfig = {};
+
+export default config;
+```
+
+4. Create the default route at the path `src/app/[[...path]]/page.tsx`. Use the `TemplateRenderer` and `EditorBlocksRenderer` to create the route.
+
+```typescript
+import { TemplateRenderer } from '@snapwp/next';
+import { EditorBlocksRenderer } from '@snapwp/blocks';
+
+export default function Page() {
+	return (
+		<TemplateRenderer>
+			{ ( editorBlocks ) => {
+				return <EditorBlocksRenderer editorBlocks={ editorBlocks } />;
+			} }
+		</TemplateRenderer>
+	);
+}
+
+```
+
+5. Create the root layout to load global styles, fonts and scripts from WP server. Make a file `src/app/layout.tsx`
+
+```typescript
+import { RootLayout } from '@snapwp/next';
+
+export default function Layout( { children }: { children: React.ReactNode } ) {
+	return (
+		<RootLayout>
+			<>{ children }</>
+		</RootLayout>
+	);
+}
+```
+
+6. Update the nextjs config. Rename the file to `next.config.mjs` so that top-level await can be used.
```

Comment:
This feels to me like it should be after step 3, to keep the config steps together